Source/Softswitch/src/softswitch_common.cpp

```diff
@@ -633,16 +656,35 @@ inline uint32_t softswitch_onRTS(ThreadCtxt_t* ThreadContext, devInst_t* device)
             {
                 // Get a pointer to the active pin
                 outPin_t* output_pin = &device->outputPins[pin];
-
+#ifdef BUFFERING_SOFTSWITCH
+                uint32_t buffRem;
+                buffRem = (ThreadContext->rtsEnd >= ThreadContext->rtsStart)   ?
+                        (((ThreadContext->rtsBuffSize - ThreadContext->rtsEnd) +
+                                    ThreadContext->rtsStart))                  :
+                            (ThreadContext->rtsStart - ThreadContext->rtsEnd);
```

Comment:
I'm not sure, but is this trying to calculate the "amount" (?) of space in the (circular) message buffer? Maybe a comment would help here.

Reply:
Yes it is, I will add a comment.